deep_quoridor/test/quoridor_env_test.py

```diff
@@ -68,16 +68,16 @@ def parse_board(board):
 
                 if ch == "v" or ch == "|":
                     forbidden_walls.append((row_n, col_n - 1, 0))
-                    if ch == "|" and not env.is_wall_between(row_n, col_n - 1, row_n, col_n):
-                        env.place_wall("player_0", (row_n, col_n - 1), 0)
+                    if ch == "|":
+                        game.step(WallAction((row_n, col_n - 1), WallOrientation.VERTICAL))
                     continue
 
                 if ch == "*":
                     potential_moves.append((row_n, col_n))
                 elif ch == "1":
-                    env.positions["player_0"] = row_n, col_n
+                    game.player_positions[0] = (row_n, col_n)
```

Comment:
here it's not setting the player position on the CombinedGrid, that's why I want it to be encapuslated :)